chpl/chpl-resources/src/test/resources/META-INF/persistence.xml

```diff
@@ -148,7 +149,7 @@
             <property name="hibernate.format_sql" value="false" />
             <property name="hibernate.driver_class" value="org.postgresql.Driver" />
             <property name="hibernate.connection.datasource"
-                value="java:comp/env/jdbc/openchpl_test" />
+                value="java:comp/env/jdbc/openchpl" />
```

Comment:
Shouldn't the test system connect to the test database?

Reply:
Yes, leftover from testing against all the listing activity. Addressed.